### PR TITLE
BioRxiv Harvester & Normalizer

### PR DESCRIPTION
**Pull request by leb2dg**
Rss feed

### CODE REVIEW

**Comment by leb2dg** [comment]
@chrisseto, this ready for review! There is no SSL :/

File: providers/org/biorxiv/harvester.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        'ns0': 'http://purl.org/rss/1.0/',
+        'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
+    }
+
+    url = 'http://connect.biorxiv.org/biorxiv_xml.php'
+
+    def do_harvest(self, start_date, end_date):
+        # BioRxiv does not have filter dates; returns 30 most recent
+        start_date = start_date.date()
+        url = (furl(self.url).set(query_params={
```
Comment (chrisseto) [inline providers/org/biorxiv/harvester.py]: No need for the extra `(``)`s around `furl(...)`

File: providers/org/biorxiv/normalizer.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class Contributor(Parser):
+    order_cited = ctx['index']
+    person = ctx
+    cited_name = ctx
+
+
+class CreativeWork(Parser):
+    title = ctx.item('dc:title')
```
Comment (chrisseto) [inline providers/org/biorxiv/normalizer.py]: What happens with the `<![CDATA[` junk here?

Comment (leb2dg) [inline providers/org/biorxiv/normalizer.py]: It is not included in the data which is awesome.

**Comment by leb2dg** [comment]
@chrisseto, comments addressed 🎉

**Pull request merged**